Framework/PythonInterface/mantid/geometry/src/Exports/ComponentInfo.cpp

```diff
@@ -0,0 +1,146 @@
+#include "MantidGeometry/Instrument/ComponentInfo.h"
+#include "MantidKernel/Quat.h"
+#include "MantidKernel/V3D.h"
+#include <boost/python/class.hpp>
+#include <boost/python/copy_const_reference.hpp>
+#include <boost/python/list.hpp>
+#include <boost/python/return_value_policy.hpp>
+
+using Mantid::Geometry::ComponentInfo;
+using Mantid::Kernel::Quat;
+using Mantid::Kernel::V3D;
+using namespace boost::python;
+
+// Helper function for converting std::vector<size_t> to a Python list
```

Comment:
Remind me, what is the reason requireing this conversion?

Reply:
How is this solved in other parts of Mantid?

Reply:
I think there might be something in Converters. I'll have a look and change these methods to use them.